### ISSUE TITLE
Incorrect render of Saved/Bookmarked button in the reading list

### ISSUE THREAD

**Issue by imadityang** [thepracticaldev/dev.to#1347]
**Describe the bug**
clicking the "LOAD MORE POSTS" button in the readinglist, the "SAVED" bookmarks button after each article becomes "SAVE" showing as if posts are not bookmarked. See screenshots below

**To Reproduce**

1. Go to your reading list - [here](https://dev.to/readinglist)
2. Now below every post you should see a "SAVED" button indicating post is bookmarked.
3. Scroll down and click "LOAD MORE POSTS"
4. Now, below every post you will see a "SAVE" button as if posts are not bookmarked.

**Screenshots**
- Normal Reading list:
![annotation](https://user-images.githubusercontent.com/10151103/50090447-90a51580-022e-11e9-9021-16086f5650f8.png)

- When i click "LOAD MORE POSTS" button:
![annotation1](https://user-images.githubusercontent.com/10151103/50090448-90a51580-022e-11e9-9b6d-57a9400a0616.png)

**Desktop (please complete the following information):**

- OS: Windows 10
- Browser: Firefox(No Extensions)

**Additional context**
Normal button:

```html
<button class="article-engagement-count bookmark-engage selected">
```

When the "LOAD MORE POSTS" button is clicked, the class `selected` class is removed making button: 

```html
<button class="article-engagement-count bookmark-engage">
```

**Comment by Zhao-Andy** [comment]
Hey @imadityang, thanks for the bug report!

**Comment by mariocsee** [comment]
I think this issue may have been solved by PR #1927, which closed other reading list issues #1395 and #1785. Currently, I can't recreate this on my reading list with Chrome, Firefox, and Safari on MacOS Mojave. Are you still experiencing this, @imadityang?

**Comment by imadityang** [comment]
@mariocsee Yes it's solved now!
